src/components/HomePage.jsx

```diff
@@ -1,11 +1,35 @@
-import React from "react";
-import { Container } from "semantic-ui-react";
+import React, { useState, useEffect } from "react";
+import { Grid } from "semantic-ui-react";
+import { articlesIndex } from "../modules/apiHelper";
+import ArticleItem from "./ArticleItem";
 
 const HomePage = () => {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    (async () => {
+      const articlesResponse = await articlesIndex();
+      setArticles(articlesResponse);
+    })();
+  }, []);
+
+  let articleList = articles.map((articleItem, index) => {
+    return (
+      <ArticleItem
+        articleItem={articleItem}
+        index={index}
```

Comment:
Why do you need to pass in this value as a propp?

Reply:
This has been changed!